### DISCUSSION TITLE
Disabling ESLint's general parsing errors

### DISCUSSION THREAD

**Discussion by coordinath** [eslint/eslint#18922] [Q&A]
Hello! I am using ESLint with TypeScript (including the typescript-eslint plugin), with VSCode as my IDE. I was wondering if there is a way to disable ESLint's general parsing errors--TypeScript already flags these parsing errors and does so with more accurate source references. 
![image](https://github.com/user-attachments/assets/fee0de23-bd52-4cd8-9b90-13ba16efcad3)

I don't want to disable ESLint altogether, just these syntax warnings that my TypeScript compiler already flags anyway. Is there any way to do this?

**Comment by nzakas** [comment]
You cannot disable parsing errors from ESLint.
